app/src/main/kotlin/com/wire/android/ui/home/conversations/search/SearchAllPeopleScreen.kt

```diff
@@ -89,51 +101,69 @@ fun SearchAllPeopleScreen(
 @Composable
 private fun SearchResult(
     searchQuery: String,
-    searchResult: Map<SearchResultTitle, ContactSearchResult>,
-    contactsAddedToGroup: List<Contact>,
-    onAddToGroup: (Contact) -> Unit,
-    onRemoveContactFromGroup: (Contact) -> Unit,
+    contactsSearchResult: ImmutableList<Contact>,
+    publicSearchResult: ImmutableList<Contact>,
+    isLoading: Boolean,
+    isSearchActive: Boolean,
+    contactsAddedToGroup: ImmutableSet<Contact>,
+    onChecked: (Boolean, Contact) -> Unit,
     onOpenUserProfile: (Contact) -> Unit,
-    onAddContactClicked: (Contact) -> Unit,
+    sendConnectionRequestViewModel: SendConnectionRequestViewModel = hiltViewModel(),
     lazyListState: LazyListState = rememberLazyListState()
 ) {
     val searchPeopleScreenState = rememberSearchPeopleScreenState()
     val context = LocalContext.current
+    val snackbarHostState = LocalSnackbarHostState.current
+    val scope = rememberCoroutineScope()
 
+    val sendConnectionRequest: (UserId) -> Unit = remember {
+        { userId: UserId ->
+            val defrredResult = sendConnectionRequestViewModel.addContact(userId)
+            defrredResult.invokeOnCompletion { throwable ->
+                if (throwable != null) {
+                    return@invokeOnCompletion
+                }
+                if (defrredResult.getCompleted()) {
+                    scope.launch {
+                        snackbarHostState.showSnackbar(context.getString(R.string.connection_request_sent))
+                    }
+                }
+            }
+        }
+    }
     Column {
         LazyColumn(
             state = lazyListState,
             modifier = Modifier
+                .fillMaxHeight()
                 .weight(1f),
         ) {
-            searchResult.forEach { (searchTitle, result) ->
-                when (result) {
-                    is ContactSearchResult.ExternalContact -> {
-                        externalSearchResults(
-                            searchTitle = context.getString(searchTitle.stringRes),
-                            searchQuery = searchQuery,
-                            contactSearchResult = result,
-                            showAllItems = searchPeopleScreenState.publicResultsCollapsed,
-                            onShowAllButtonClicked = searchPeopleScreenState::toggleShowAllPublicResult,
-                            onOpenUserProfile = onOpenUserProfile,
-                            onAddContactClicked = onAddContactClicked
-                        )
-                    }
+            if (contactsSearchResult.isNotEmpty()) {
+                appLogger.d("cccc contactsSearchResult: ${contactsSearchResult.size}")
```

Comment:
nitpic: Maybe just removing the `cccc` from the logs? Or adding something like:

```suggestion
                appLogger.d("Contacts Search Result Size: ${contactsSearchResult.size}")
```